pages/index.js

```diff
@@ -30,7 +30,7 @@ export default Home
 
 const HomeHero = () => {
   return (
-    <section className="full-width-bg hero h-[60vh] bg-main-blurple text-center text-white">
+    <section className="full-width-bg hero h-[80vh] bg-main-blurple pt-[var(--header-area)] text-center text-white">
```

Comment:
does this seem like an appropriate way to get the header to interact with the illo BGs below it? the header has a negative margin bottom to pull sections below at the same time https://github.com/oakstudios/joinmastodon/pull/25/files#diff-0d12cdbdd80599cc94e30507d66a26b558ffbc42e0a3b161d1440afad9b72e6bR212